### PR TITLE
Models for network policy recipes

### PR DESCRIPTION
**Pull request by Vasundharashukla**
1. **What this PR does / why we need it**:

Models for network policy recipes:

- allow-all
- allow-all-to-pods
- deny-all-to-pods
- custom-egress
- custom-ingress

The recipes: `allow-all-egress`, `deny-all-egress` and `deny-all` do not have a specific model.

### CODE REVIEW

**Review by shreyassreenivas — APPROVED** [review]

**Review by ishangupta-ds — APPROVED** [review]

File: internal/models/policy/recipe/network/allow_all.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+*/
+
+package policyrecipenetworkmodel
+
+import "github.com/go-openapi/swag"
+
+// VmwareTanzuManageV1alpha1CommonPolicySpecNetworkV1AllowAll allow-all schema
+//
+// swagger:model VmwareTanzuManageV1alpha1CommonPolicySpecNetworkV1AllowAll
+type VmwareTanzuManageV1alpha1CommonPolicySpecNetworkV1AllowAll struct {
```
Comment (shreyassreenivas) [inline internal/models/policy/recipe/network/allow_all.go:14]: Lets start using smallnames, the model names are getting too long

Comment (Vasundharashukla) [reply internal/models/policy/recipe/network/allow_all.go:14]: Is `V1alpha1CommonPolicySpecNetworkV1AllowAll ` better?

Comment (shreyassreenivas) [reply internal/models/policy/recipe/network/allow_all.go]: Yes, thats sounds better

Comment (Vasundharashukla) [reply internal/models/policy/recipe/network/allow_all.go]: Done

**Review by shreyassreenivas — APPROVED** [review]

**Pull request merged**